modules/infra/submodules/storage/outputs.tf

```diff
@@ -31,5 +35,9 @@ output "info" {
       container_registry = join("/", concat(slice(split("/", aws_ecr_repository.this["environment"].repository_url), 0, 1), [var.deploy_id]))
       iam_policy_arn     = aws_iam_policy.ecr.arn
     }
+    costs = {
+      storage_enabled = var.domino_cost.storage_enabled
+      costs_bucket    = aws_s3_bucket.costs[0].bucket
```

Comment:
I don't think this should need the separate output. There is an output that outputs all s3 buckets based on the local variable that's in `storage/main.tf`, so we can determine both the name of the costs bucket via that as well as whether it was provisioned by whether the value exists.

Reply:
thanks, will update.